lib/rack/user_agent/detector.rb

```diff
@@ -32,10 +36,18 @@ def from_android_tablet?
         os == "Android" && !android_mobile?
       end
 
+      def from_android_os?
+        from_android? || from_android_tablet?
+      end
+
       def from_windows_phone?
         os == "Windows Phone OS"
       end
 
+      def from_windows_phone_os?
+        from_windows_phone?
```

Comment:
@k0kubun 
You are right, it was useless :)
I just added this for detecting Windows Phone and other Windows Phone devices such as tablets, but it seems that *Windows Phone Tablet* doesn't exist. I got rid of this 🚮